### ISSUE TITLE
Sync error - missing last name

### ISSUE THREAD

**Issue by Danman7** [FundacionParaguaya/MentorApp#1092]
We have a sync error with the server message `El apellido del primer miembro no puede ser nulo` and a draft where the family participant has no last name. There are 11 such events. https://app.bugsnag.com/penguin-digital/stoplight-mobile/errors/5d6e6f754b2fa10019d03631?filters[event.since][0]=30d&filters[error.status][0]=open

**Comment by Danman7** [comment]
This is most probably the session: https://app.testfairy.com/projects/6819694-stoplightmobile/builds/8951877/sessions/4434114639/?accessToken=VueqTcw1ccTpkU6IN6S/1woVERs

**Comment by Danman7** [comment]
We spent a working day worth of time on this issue alone. Luckily it's related to only a single lifemap.

First we identified the issue to be a missing last name, but that was only the message from the server. The participant was actually missing doc type and number as well. His data looked much like a member's data, not a participant.

Then we found two session the device the report came from. One is from before the problem occured - the client inspects his app and everything is synced. 0 errors. The next session we have has the error already, so we don't have a session from when it was introduced. In this session the client wanders around the draft, going back to the first screen and there actually seen the empty fields and pressing continue. The form validation works fine for him and he re-enters the data.

We could not find any specific piece of code that can remove a participant's last name or were able to reproduce the issue. We will remove some failsafes that were modifying family data, introduced for an old Geco issue so they can sync. This should be a failsafe for any future scenarios.
